builders/interoperability/xcm/xcm-sdk/xcm-sdk.md

```diff
@@ -875,18 +875,21 @@ console.log(isXcmSdkDeposit(deposit)) // Returns false
 
 #### Convert Balance to Decimal or BigInt {: #decimals }
 
-To convert a balance to decimal format, you can use the `toDecimal` function which returns a given number in decimal format based on the number of decimals provided. You can optionally pass in a value a third argument to dictate the maximum number of decimal places used, otherwise the default is `6`.
+To convert a balance to decimal format, you can use the `toDecimal` function which returns a given number in decimal format based on the number of decimals provided. You can optionally pass in a value a third argument to dictate the maximum number of decimal places used, otherwise the default is `6`; and a fourth argument which dictates the [rounding method](https://mikemcl.github.io/big.js/#rm) of the number.
+The `toDecimal` function returns a `Big` number type that you can convert to number or string using its methods `toNumber`, `toFixed`, `toPrecision` and `toExponential`. We recomend using them as `string` since big numbers or numbers with a lot of decimals can lose precision using `number` types
```

Comment:
```suggestion
The `toDecimal` function returns a Big number type that you can convert to a number or string using its methods `toNumber`, `toFixed`, `toPrecision`, and `toExponential`. We recommend using them as a string, since big numbers or numbers with a lot of decimals can lose precision when using number types.
```